index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="stylesheet/main.css">
+    <link rel="stylesheet" href="stylesheet/index.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Playfair+Display:ital,wght@0,400..900;1,400..900&display=swap" rel="stylesheet">
+</head>
+<body class="main">
+<div class="main--flex-containers">
+    <div class="main--banner-container">
+        <img class="main--banner-img" src="images/banners/homepage_01.jpg" alt="">
+        <header class="header header--index">
+            <a href=""><img class="logo logo-main" src="images/logos/logo copy.png" alt=""></a>
+            <nav class="nav nav--main">
+                
+                <ul class="nav--ul nav--ul-main">
```

Comment:
`nav--ul nav--ul-main` nu cred ca e nevoie de ambele clase .. a 2-a din cate vad nici nu e folosita 

si as mai sugera sa nu legi elementul de HTML de clasa din CSS ... ca sa ai posibilitatea sa schimbi in viitor fara sa atingi CSS-u 

eg: `nav`